Gemfile

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-ruby '2.4.1'
+ruby '>= 2.3.0', '< 2.5.0'
```

Comment:
This is probably fine, because we can continue to use `.ruby-version` to mean "the version we expect developers to be working against", but relaxing in `Gemfile` will allow admins who can't run very latest and don't have access to rvm/rbenv/etc to use whatever they have in some supported range.